reference/docs-conceptual/windows-powershell/install/Installing-the-Windows-PowerShell-2.0-Engine.md

```diff
@@ -90,23 +90,33 @@ minimum. This requirement is fulfilled by Microsoft .NET Framework 3.5.
   PowerShell 2.0 Engine**.
 
 For information about starting the Windows PowerShell 2.0 Engine, see
-[Starting the Windows PowerShell 2.0 Engine](../Starting-the-Windows-PowerShell-2.0-Engine.md).
+[Starting the Windows PowerShell 2.0 Engine][01].
 
 ## On Earlier Systems
 
-The [Windows Management Framework 4.0](https://www.microsoft.com/download/details.aspx?id=40855)
-package that installs Windows PowerShell 4.0 on Windows 7, Windows Server 2008 R2, and Windows
-Server 2012, includes the Windows PowerShell 2.0 Engine. The Windows PowerShell 2.0 Engine is
-enabled and ready to use, if necessary, without additional installation, setup, or configuration.
+The Windows Management Framework 4.0 package that installs Windows PowerShell 4.0 on Windows 7,
+Windows Server 2008 R2, and Windows Server 2012, includes the Windows PowerShell 2.0 Engine. The
```

Comment:
I think this might need rewriting for `5.1` since we're marking `4.0` as unsupported - we could mention that it comes installed on 2012R2 and 8.1 instead?

Thinking about this whole section, I don't believe there's _any_ supported OS that doesn't ship with a version of Windows PowerShell that includes the 2.0 Engine.

I'm also wondering if we might need to rewrite to not address Operating System versions that are out of extended support.

Reply:
Removed